Modules/Filtering/Denoising/include/itkPatchBasedDenoisingImageFilter.hxx

```diff
@@ -390,7 +390,7 @@ PatchBasedDenoisingImageFilter<TInputImage, TOutputImage>::EnforceConstraints()
     if (m_ImageMax[ic] <= m_ImageMin[ic])
     {
       itkExceptionMacro(<< "Each image component must be nonconstant.  "
-                        << "Component " << ic << " has the constant value " << m_ImageMax[ic] << ".\n";);
+                        << "Component " << ic << " has the constant value " << m_ImageMax[ic] << ".\n");
```

Comment:
I don't think it's necessary for an exception message to end with "\n" either, but I wasn't sure either if removing the "\n"  would make some users unhappy..., what do you think?

Reply:
You need to look at how formatted messages look on the console before deciding whether to remove `\n`.

Reply:
> You need to look at how formatted messages look on the console before deciding whether to remove \n.

OK, thanks @dzenanz, then I prefer not to decide, and leave the \n as they are, for now.  The semi-colons are obviously mistaken anyway!